Jenkinsfile_nightly

```diff
@@ -12,5 +12,8 @@ def component = "end-to-end-tests"
 
 withNightlyPipeline(type, product, component) {
   enableFullFunctionalTest()
+  after('fullFunctionalTest') {
+    steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'functional-output/**/*'
```

Comment:
we may need a folder in our repo `functional-output` ?

Reply:
I added it. Added `.gitignore` file to the `functional-output` directory because the empty directory is not added to the repo

Reply:
see `add-functional-output` branch
https://github.com/hmcts/bulk-scan-end-to-end-tests/tree/add-functional-output